portfolio/src/main/webapp/style.css

```
@@ -7,19 +7,25 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+
 #quote-container {
```

Sorry I meant just one newline in between:
```suggestion
}

#quote-container {
```